packages/fxa-settings/src/components/OAuthDataError/index.tsx

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { AuthError } from '../../lib/oauth';
+import AppLayout from '../AppLayout';
+import Banner, { BannerType } from '../Banner';
+import CardHeader from '../CardHeader';
+
+const OAuthDataError = ({ error }: { error: AuthError }) => {
+  return (
+    <AppLayout>
+      <CardHeader
+        headingText="Bad Request"
+        headingTextFtlId="error-bad-request"
+      />
+      {/* TODO: This error is edge casey, unlocalized in Backbone, and unlikely
```

Comment:
This would mean adding the errors returned in `checkOAuthData` to the new `oauth/en.ftl` file and using `getLocalizedError` here. I've done the prework for it since I needed to localize the "Try again" error, so I can file this as a 1-pointer follow up or remove this comment.

Reply:
Good point, will do.